File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/cake/CakeCreateParam.java
package com.cakk.domain.mysql.dto.param.cake;

import java.util.List;

import com.cakk.domain.mysql.entity.cake.Cake;
import com.cakk.domain.mysql.entity.cake.CakeCategory;
import com.cakk.domain.mysql.entity.user.User;

public record CakeCreateParam(
	Cake cake,
	List<CakeCategory> cakeCategories,
	List<String> tagNames,
	User owner,
	Long cakeShopId
) {
}

File: cakk-core/src/main/kotlin/com/cakk/core/dto/param/cake/CakeCreateParam.kt
package com.cakk.core.dto.param.cake

import com.cakk.infrastructure.persistence.entity.cake.CakeCategoryEntity
import com.cakk.infrastructure.persistence.entity.cake.CakeEntity
import com.cakk.infrastructure.persistence.entity.user.UserEntity


data class CakeCreateParam(
	val cake: CakeEntity,
	val cakeCategories: List<CakeCategoryEntity>,
	val tagNames: List<String>,
	val owner: UserEntity,
	val cakeShopId: Long
)
